restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/configuration/How to get started with Restcomm-Connect and Mysql.adoc

```diff
@@ -41,19 +43,21 @@ image::images/13.png[]
 If the process was successful you will see the following:
 
 image::images/14.png[]
-*Step 3 – Edit the restcomm.xml file to point the DAO to MySQL*
+== Edit the restcomm.xml file to point the DAO to MySQL
 
 * Go to the *$RESTCOMM_HOME/standalone/deployments/restcomm.war/WEB-INF/conf/*.
 Open the *restcomm.xml* file.
 * Find the dao-manager tag and change the sql-files path to mariadb as shown below:
+
+[source,bash]
 ----
 <dao-manager class="org.mobicents.servlet.restcomm.dao.mybatis.MybatisDaoManager">
   <configuration-file>${restcomm:home}/WEB-INF/conf/mybatis.xml</configuration-file>
   <data-files></data-files>
   <sql-files>${restcomm:home}/WEB-INF/scripts/mariadb/sql</sql-files>
 </dao-manager>
 ----
-*Step 4 – Download Mysql Java Client Driver*
+== Download Mysql Java Client Driver
 
 * Download the latest MySQL Java Connector from http://mvnrepository.com/artifact/mysql/mysql-connector-java.
 * Put the jar file in
```

Comment:
At the end of the article, we should probably link to start restcomm guide and also show how it can be verified that the new setup was successful